hg-fast-export.py

```diff
@@ -51,8 +51,7 @@ def wr_no_nl(msg=b''):
     stdout_buffer.write(msg)
 
 def wr(msg=b''):
-  wr_no_nl(msg)
-  stdout_buffer.write(b'\n')
+  wr_no_nl(msg + b'\n')
```

Comment:
My Python intuition is not developed enough to know if two writes is more or less efficient than doing an append when the prefix is large. Do you know?

Reply:
Should make no difference. In Python 3 `stdout_buffer` is a BufferedWriter anyway, so `write()` is not actually writing, just adding to the buffer.